packages/knip/src/plugins/netlify/helpers.ts

```diff
@@ -0,0 +1,11 @@
+import type { FunctionsConfig } from './types.js';
+
+export const validFunctionExtensions = () =>
+  ['.js', '.mjs', '.cjs', '.ts', '.mts', '.cts'].map(ext => ext.slice(1)).join(',');
```

Comment:
Sorry for the nitpicking, but this looks like it could just as well be literal strings/globs in `PRODUCTION_ENTRY_FILE_PATTERNS` directly?